os/Kconfig

```diff
@@ -176,6 +176,7 @@ config BUILD_FLAT
 config APP_BINARY_SEPARATION
 	bool "Application binary separation build"
 	select BUILD_PROTECTED
+	select BINFMT_LOADABLE
```

Comment:
This should not be selected by app_bin_separation. This is not done by ELF. It's right.